alert-database/src/test/java/com/synopsys/integration/alert/database/api/DefaultCustomCertificateAccessorTest.java

```diff
@@ -0,0 +1,174 @@
+package com.synopsys.integration.alert.database.api;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.alert.common.exception.AlertDatabaseConstraintException;
+import com.synopsys.integration.alert.common.persistence.model.CustomCertificateModel;
+import com.synopsys.integration.alert.database.api.mock.MockCustomCertificateRepository;
+import com.synopsys.integration.alert.database.certificates.CustomCertificateRepository;
+
+public class DefaultCustomCertificateAccessorTest {
+    private final String alias = "alias-test";
+    private final String content = "content-test";
+
+    private CustomCertificateModel expectedCustomCertificateModel = new CustomCertificateModel(alias, content, new Date().toString());
```

Comment:
If we want to compare the updated times, we should move the Date to its own field. And in the tests we can create a new Date if we want to use a newer time, and use the Date field when they should be the same.